src/main/java/com/xeeva/catalog/RequisitioningPage.java

```diff
@@ -421,38 +428,78 @@ public void click_FavoriteItem(){
 		lnkShowFavItems.click();
 		driver.setPageTimeout(3);
 	}
-	
-	
-	  
-	  /**
-	   * @summary: Method to get the Item Number from catalog table.
-	   * @author praveen namburi,@version: Created 13-09-2016
-	   * @return getItemNumber
-	   */
-	  public String  getItemNumberFromCatalog(){
-		  driver.setPageTimeout(2);
-		  String getItemNumber = null;
-		  String getLocalItemsCount = lblLocalItems.getText();
-		  TestReporter.log("Local-Items Count is: "+ getLocalItemsCount);
-		  
-		  if(!getLocalItemsCount.contains("0")){
-				TestReporter.log("Click on Local-Items tab.");
-				LocalItemsTab localItem = new LocalItemsTab(driver);
-				localItem.click_localItemsTab();
-				List<WebElement> localItems = localItemsGrid;
-				for(WebElement inputItem : localItems){
-					String itemNumber = inputItem.getText().trim();
-					getItemNumber = itemNumber;
-					break;
-				}
-		  }else if(getLocalItemsCount.contains("0")){
-				TestReporter.assertTrue(false," 'No Records Found !!' in Local Items tab. ");
-		  }
+
+
+
+	/**
+	 * @summary: Method to get the Item Number from catalog table.
+	 * @author praveen namburi,@version: Created 13-09-2016
+	 * @return getItemNumber
+	 */
+	public String  getItemNumberFromCatalog(){
+		driver.setPageTimeout(2);
+		String getItemNumber = null;
+		String getLocalItemsCount = lblLocalItems.getText();
+		TestReporter.log("Local-Items Count is: "+ getLocalItemsCount);
+
+		if(!getLocalItemsCount.contains("0")){
+			TestReporter.log("Click on Local-Items tab.");
+			LocalItemsTab localItem = new LocalItemsTab(driver);
+			localItem.click_localItemsTab();
+			List<WebElement> localItems = localItemsGrid;
+			for(WebElement inputItem : localItems){
+				String itemNumber = inputItem.getText().trim();
+				getItemNumber = itemNumber;
+				break;
+			}
+		}else if(getLocalItemsCount.contains("0")){
+			TestReporter.assertTrue(false," 'No Records Found !!' in Local Items tab. ");
+		}
 		return getItemNumber;
 	  
 	  }	
 	  
+	  /**
+	   * @summary: Method to cancel the requisition record and verify them.
+	   * @author praveen namburi, @Version: Created 23-09-2016
+	   * @param comments
+	   */
+	  public void cancelRequisitionFromRecentOrders(String comments){
+		  tblRecentOrdersGrid.syncVisible();
+		  driver.setPageTimeout(5);
+		  btnSeeMore.syncEnabled(5);
+		  //btnSeeMore.click();
+		  driver.executeJavaScript("arguments[0].click();", btnSeeMore);
+		  driver.setPageTimeout(5);
+		  List<WebElement> getRows = tblMainRecentOrdersGrid;
+		  int rowsCount = getRows.size();
+		  TestReporter.log("Total rows in RecentOrders Grid table: "+ rowsCount);
+			  
+		    for(int row=1; row<=rowsCount; row++){
+				String getStatus = driver.findElement(By.xpath("//*[@id='gvRecentOdersGrid']/tbody/"
+						+ "tr["+ row +"]/td[8]/span")).getText();
+				if(!getStatus.contains("Canceled By Requester")){
+					driver.setPageTimeout(2);

```

Comment:
Please remove.  If you're wanting to change the element time out for the next statement, you can use driver.SetElementTimeout(2) but after you do the next line, need to return to the default driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);